myscripts.js

```diff
@@ -0,0 +1,9 @@
+window.addEventListener('load',()=>{
+    const navList=document.querySelectorAll("#nav__list li");
+    const emptiness=document.querySelector(".empty_area");
+  for(let i=0;i<navList.length;i++){
+    navList[i].onclick=function(){
+        document.getElementById("txt").value=this.innerText;
+    };
+  };
```

Comment:
Why did you remove the previous function?

It is always better to use JS native functions, instead of manually typing for loop.